VirtualTest/VirtualFunctions/VirtualFunctions.h

```diff
@@ -0,0 +1,278 @@
+#include "Map.h"
+
+uint8_t iSize = 11, iRow = 1, iColumn = 1;
+char virtualMap[11][11] = { {'.','W','.','W','.','W','.','W','.','W','.'},
+                            {'W','R','W','B','W',' ','W',' ','.','2','W'},
+                            {'.','.','.','.','.','.','.','.','.','.','.'},
+                            {'W',' ','.',' ','.',' ','.',' ','W',' ','W'},
+                            {'.','W','.','.','.','W','.','.','.','.','.'},
+                            {'W','B','W','3','W',' ','W',' ','.',' ','W'},
+                            {'.','.','.','.','.','W','.','W','.','.','.'},
+                            {'W',' ','.',' ','.','1','.','1','.',' ','W'},
+                            {'.','W','.','.','.','.','.','.','.','.','.'},
+                            {'W',' ','W',' ','W',' ','.',' ','W',' ','W'},
+                            {'.','W','.','W','.','W','.','W','.','W','.'} };
+
```

Comment:
const

Reply:
also, every proper constant (like this one, not in the parameters) must be in the format of:
char kVirtualMap;

Reply:
Ok, I did not know that. But sometimes I do change virtualMap to get visual feedback of the actual position of the robot and be able to compare it with the position in the tiles_map.